practice/src/css/sections/discount.css

```diff
@@ -0,0 +1,114 @@
+.wrapper-discount {
+  padding-bottom: 50px;
+  border-radius: 70px 20px 20px 20px;
+  background: var(--dark-blue-background-color) url(../../assets/images/discount/map-mobile.png) no-repeat center / contain;
+}
+
+.discount-section {
+  padding-top: 40px;
+}
+
+.discount-heading {
+  text-align: center;
+  padding-bottom: 99px;
+}
+
+.btn-claim {
+  font-size: 14px;
+  background-color: var(--tertiary-background-color);
+}
+
+.discount-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 50px;
+}
+
+.icon-heart {
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: translateY(-50%);
+}
+
+.discount-comment {
+  display: none;
+}
+
+.wrapper-discount {
+  position: relative;
+}
+
+/* Tablet */
+@media (min-width: 768px) {
+  .rojali-info {
+    font-size: var(--font-size-xsmall);
+    line-height: 21px;
+    font-weight: var(--font-weight-regular);
+    color: var(--dark-text-color);
+  }
+
+  .comment-customer {
+    font-size: var(--font-size-small);
+    font-weight: var(--font-weight-medium);
+    line-height: 23px;
+    color: var(--dark-text-color);
+  }
+
+  .wrapper-discount {
+    background-image: none;
+    background-color: var(--primary-background-color);
+  }
+
+  .discount-main {
+    background: var(--dark-blue-background-color) url(../../assets/images/discount/map-desktop.png) no-repeat;
+    border-radius: 70px 20px 20px 20px;
+    padding-bottom: 50px;
+  }
+
+  .discount-title {
+    width: 500px;
+  }
+
+  .discount-description {
+    width: 341px;
+    color: var(--dark-text-color);
+  }
+
+  .discount-heading {
+    padding-bottom: 30px;
+    width: 461px;
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    align-items: center;
+  }
+
+  .icon-heart {
+    transform: translate(-10%, -50%);
+  }
+
+  .discount-comment {
+    display: block;
+    position: absolute;
```

Comment:
why do you need to use position absolute here?

Reply:
I want to set its position based on the parent element which is .wrapper-discount with position: relative